From bea7f6965378c45f0b543bcd65f579fa3b9e0ef2 Mon Sep 17 00:00:00 2001
From: ran shalit <rshalit@habana.ai>
Date: Thu, 22 Dec 2022 15:16:49 +0200
Subject: [PATCH] [SW-66343] gaudi3: remove lkd_dev_spec_reg

lkd_dev_spec_regs is a weak function used for updating
cpu_dyn_regs, which are passed to host as part of COMMs
protocol in preboot.
In gaudi3 this function is implemented in preboot-rtos,
and not relevant in legacy preboot. Hence, function removed.

Change-Id: Id56c7ab9ba6d37ffdad1ecd6d25d8f7befb70ebb
Signed-off-by: ran shalit <rshalit@habana.ai>
---

diff --git a/arch/gaudi3/arc/common.c b/arch/gaudi3/arc/common.c
index 0dae29b..b60b0d2 100644
--- a/arch/gaudi3/arc/common.c
+++ b/arch/gaudi3/arc/common.c
@@ -645,11 +645,6 @@
 	cpu_boot_dev_sts_set(CPU_BOOT_DEV_STS_FW_HARD_RST_EN);
 }
 
-void lkd_dev_spec_regs(struct cpu_dyn_regs *regs)
-{
-	/* TODO: add cpu_dyn_regs asigments */
-}
-
 bool preboot_is_valid_address_range(u64 addr, int len)
 {
 	if (addr >= SP_SRAM_BASE &&
